VKApp/VKApp/Groups/GroupsTableViewController.swift

```diff
@@ -28,17 +28,22 @@ final class GroupsTableViewController: UITableViewController {
 
     // MARK: - Private Property
 
-    private var groups = [
-        Group(name: Constants.oneGroupTitleText, imageName: Constants.imageName),
-        Group(name: Constants.twoGrroupTitleText, imageName: Constants.twoImageName),
-        Group(name: Constants.threeGroupTitleText, imageName: Constants.threeImageName),
-    ]
+    private var groupsResponse: Group?
+    private var groupsTwo: [GroupItem] = []
```

Comment:
```suggestion
    private var: [GroupItem] = []
```
надо дать более подходящее имя